project-base/storefront/cypress/support/index.ts

```diff
@@ -191,6 +200,34 @@ const removeBlackoutsAfterScreenshot = () => {
     });
 };
 
+const removePointerEventsBeforeScreenshot = (removePointerEvents: (TIDs | string)[]) => {
+    cy.document().then((doc) => {
+        const style = doc.createElement('style');
+        style.setAttribute('id', 'disable-pointer-events');
+        doc.head.appendChild(style);
+
+        const selectors = removePointerEvents.map((selector) => {
+            if (Object.values<any>(TIDs).includes(selector)) {
+                return `[tid='${selector}']`;
+            }
+            return selector;
+        });
+
+        const selectorString = selectors.join(', ');
+
+        style.innerHTML = `${selectorString} { pointer-events: none !important; }`;
+    });
+};
```

Comment:
The code looks hacky, but if it solves the problem I'm fine with it.
But I have one point, for me the better solution is when we map all elements and give them style pointer-events: none !important; instead of setting it globally

Reply:
Solved on call